python/vowpalwabbit/sklearn_vw.py

```diff
@@ -12,6 +12,7 @@
 from sklearn.linear_model.base import LinearClassifierMixin, SparseCoefMixin
 from sklearn.datasets import dump_svmlight_file
 from sklearn.utils import shuffle
+from sklearn.utils._pprint import _EstimatorPrettyPrinter
```

Comment:
Also, I believe `_pprint` is meant to be a private method and should not be imported as such. I think @jackgerrits would be a better person to comment than me though.

Reply:
I think you are right that's why it is failing on Travis. But it is working on local-

```julia
In [5]: model.fit(data.x, data.y)                                                                        
Out[5]: VWClassifier(convert_to_vw=True, loss_function='logistic', passes=1, quiet=True)
```

Reply:
I believe it works because of the difference between the Python version.

Reply:
We should not depend on private methods in sklearn, they can change between versions which is exactly the issue you are seeing.

Reply:
Okay, got it. So I think I have to implement something like https://github.com/scikit-learn/scikit-learn/pull/11705 in VW and then I can use it from VW itself.